### ISSUE TITLE
"Artigos mais recentes" deve mostrar apenas documentos citáveis

### ISSUE THREAD

**Issue by alexxxmendonca** [scieloorg/opac#885]
O feed de "Artigos mais recentes" deve mostrar apenas documentos consideráveis "citáveis":

![image](https://user-images.githubusercontent.com/4059679/35271801-5be56ece-001a-11e8-84c4-ce1e8131ffee.png)

**Comment by jfunez** [comment]
Bom dia @alexxxmendonca 
Você poderia deixar explicito quando um documento é considerado "citável"?

Obg!

**Comment by alexxxmendonca** [comment]
Juan, não tenho certeza, mas existem regras já definidas para isso, que foram utilizadas no Search SciELO -- há um cluster para isso.

Tem a ver com o documento possuir título, autores, resumo, referências etc.

Dê uma olhada na regra desse cluster do Search.

**Comment by jfunez** [comment]
@jamilatta pode dar um help aqui com as regras definidas no cluster do search indicado pelo alex?

**Comment by jamilatta** [comment]
@jfunez 

Para o search os documentos consideráveis citáveis são os documento com os seguintes tipos: https://github.com/scieloorg/search-journals/blob/beta/proc/updatesearch/pipeline_xml.py#L43

Repare na linha: https://github.com/scieloorg/search-journals/blob/beta/proc/updatesearch/pipeline_xml.py#L122, caso o documento seja algum desses tipos ele é considerado citável.

**Comment by robertatakenaka** [comment]
@jamilatta os links estão quebrados.

**Comment by jamilatta** [comment]
Um documento considerável citável no search: 

https://github.com/scieloorg/search-journals/blob/master/proc/updatesearch/pipeline_xml.py#L43

Pelo que entendi um documento é citável(no contexto do search) quando tem os seguintes campo: 

```python 
CITABLE_DOCUMENT_TYPES = (
    u'article-commentary',
    u'brief-report',
    u'case-report',
    u'rapid-communication',
    u'research-article',
    u'review-article'
)
```

**Comment by robertatakenaka** [comment]
@jamilatta e @alexxxmendonca se é pelo tipo então está ótimo

**Comment by patymori** [comment]
@alexxxmendonca , poderia validar se está OK?

**Comment by alexxxmendonca** [comment]
@patymori qual ambiente? Produção ou homologação?

**Comment by patymori** [comment]
@alexxxmendonca , produção.

**Comment by alexxxmendonca** [comment]
Hummm, não sei se tenho condições de avaliar.

Como identificar no texto completo se um documento é considerável por nós citável ou não citável?

Na página do periódico Cadernos Pagu, por exemplo, os últimos documentos da lista não aparentam ser documentos citáveis e mesmo assim estão aparecendo na lista:

![image](https://user-images.githubusercontent.com/4059679/49813324-6515bd00-fd4e-11e8-94a6-7f593c2a086d.png)

https://new.scielo.br/journal/cpa/

Links dos documentos: 
https://new.scielo.br/article/cpa/2018.n54/e185400/
https://new.scielo.br/article/cpa/2018.n54/e185401/

Porém não me sinto em condições de aprovar esse ticket.

**Comment by jamilatta** [comment]
Sim, mas baseado no comentário que você fez parece que não é o suficiente :-), veja: 

```
Na página do periódico Cadernos Pagu, por exemplo, os últimos documentos da lista não aparentam ser documentos citáveis e mesmo assim estão aparecendo na lista:```

**Comment by alexxxmendonca** [comment]
De acordo com a Renata, não dá para dizer se é citável ou não "a olho nu".

Não tenho condições de validar esse ticket.

**Comment by jamilatta** [comment]
Acredito que até que tenhamos bem claro o que é um documento citável esse é desnecessário aprovar esse ticket.

**Comment by RPostalli** [comment]
Um documento citável é aquele que pra sua base tem importância para métricas, cada base de dados utiliza os tipos de documentos que querem como citáveis, com base em suas políticas.
O [Web of Science](http://ipscience-help.thomsonreuters.com/incitesLiveJCR/JCRGroup/howDoI/citableItemWoS.html) por exemplo só considera: 
Article 'research-article'
Review 'review-article'
Proceedings paper

Já SciELO Citation Index considera o mesmo que nós no Analytics
u'article-commentary',
u'brief-report',
u'case-report',
u'rapid-communication',
u'research-article',
u'review-article'

Nós consideramos todos os documentos indexados em SciELO como citáveis, segundo instruções do próprio Abel e publicado nos [Critérios SciELO Brasil](http://www.scielo.br/avaliacao/Criterios_SciELO_Brasil_versao_revisada_atualizada_outubro_20171206.pdf) item 5.2.2. (Com exceção de Errata e Retratação, documentos nascidos sem autoria e são apenas informativos.)

Com as novas regras de indexação promovidas pelos Critérios, só publicamos documentos que [obrigatoriamente possuam 5 itens](https://scielo.readthedocs.io/projects/scielo-publishing-schema/pt_BR/latest/narr/scielo-brasil.html):
1 Autoria;
2 Afiliação;
3 Título (diferente da seção);
4 Citação;
5 Referência.

Como todo boa regra, esta possui suas exceções e também foram discutidas com o Abel em reunião presencial, onde:
- Arquivo de Imagem (usado mais em revistas de ciências da saúde)
Libera sem citação

- Artigo de vídeo (usado mais em revistas de ciências da saúde)
Libera sem citação

- Entrevista
Libera sem citação 

- Tradução de artigo
Só libera publicação quando for documento histórico

Ou seja, na minha opinião nosso Analytics a partir da publicação dos novos Critérios (em 2017) deveria considerar todos os tipos indexáveis em SciELO como documentos citáveis. E por este motivo o feed site deve mostrar todos os artigos mais recentes menos Erratas e Retratações.

**Comment by RPostalli** [comment]
Apenas fica a pergunta aqui: Quem decidiu a lista a abaixo como citável para Analytics?
u'article-commentary',
u'brief-report',
u'case-report',
u'rapid-communication',
u'research-article',
u'review-article

**Comment by alexxxmendonca** [comment]
@RPostalli essa lista foi o Abel quem definiu.

**Comment by RPostalli** [comment]
@alexxxmendonca eu acho que essa definição mudou depois dos critérios. Veja Critérios:

5.2.2. Tipos de documentos
A partir de 2018, somente serão indexados, publicados e incluídos nas métricas de desempenho dos
periódicos da Coleção SciELO Brasil, documentos que apresentem conteúdo científico relevante, com
autoria e afiliação institucional dos autores, título próprio diferente do título da seção, citações e
referências bibliográficas.

O melhor seria conversar com ele.

**Comment by robertatakenaka** [comment]
@RPostalli optamos por seguir esta lista e não a dos critérios porque há artigos que poderiam não ter os 5 ítens:

- 1 Autoria;
- 2 Afiliação;
- 3 Título (diferente da seção);
- 4 Citação;
- 5 Referência.

como por exemplo os provisional pdf

O objetivo maior deste issue, é dar destaque a alguns tipos de documentos. Então, acredito que seguir esta lista é um critério aceitável.

**Comment by RPostalli** [comment]
Se o motivo é dar destaque só para artigos citáveis e o nossa política adota que 6 tipos são citáveis, adotem está regra para criar um feed automático utilizando artigos com estes @article-type.

Como disse para o @alexxxmendonca, não é possível extrair este dado a olho nu, ele está marcado no XML e não aparece na interface, ou seja, não tem como ser algo manual. A não ser que abram XML por XML para verificar o tipo de artigo marcado.
